src/main/backends/linux/kde/wayland/backend.ts

```diff
@@ -441,7 +442,10 @@ class CustomInterface extends DBus.interface.Interface {
     pointerY: number
   ) {
     if (this.wmInfoCallback) {
-      this.wmInfoCallback({ windowName, appName, pointerX, pointerY });
+      this.wmInfoCallback({ windowName, appName, pointerX, pointerY, screen.getDisplayNearestPoint({
```

Comment:
The property name is missing here. But I guess that the line will be too long afterwards. So you should run `npm run prettier` to format all code.

```suggestion
      this.wmInfoCallback({ windowName, appName, pointerX, pointerY, workearea: screen.getDisplayNearestPoint({
```